### PR TITLE
CDM-32: Create Title related components

### PR DESCRIPTION
**Pull request by danielaprado**
### Description

- Create Title component
- Create BreadcrumbsTitle component
- Create ArrowTitle component
- Create IconTitle component
- Desktop & Mobile responsive

### Attachments
- **Desktop**
image
image

- **Mobile**
Screenshot 2024-01-08 at 15 36 27
Screenshot 2024-01-08 at 15 36 58
Screenshot 2024-01-08 at 15 37 24

### Reviewers

- [x] @catarinapm 
- [x] @mariasbl

### CODE REVIEW

**Comment by mariasbl** [comment]
I think the elements in the mobile are too big

**Comment by danielaprado** [comment]
> I think the elements in the mobile are too big

Which components? All of them? 🤔

**Comment by catarinapm** [comment]
Is the icon on the Desktop with some issue related to the browser? It's smaller than the one in the mobile version  <.<

File: fe/app/components/Divider/Divider.models.ts
Comment (catarinapm) [inline fe/app/components/Divider/Divider.models.ts:1]: Why is this file here? 🥸

Comment (catarinapm) [reply fe/app/components/Divider/Divider.models.ts:1]: Got it. Is .ts instead of .tsx. You're right. Keep it

**Review by catarinapm — APPROVED** [review]

**Review by mariasbl — APPROVED** [review]
